events/migrations/0001_initial.py

```diff
@@ -0,0 +1,156 @@
+# Generated by Django 2.1.15 on 2020-07-07 15:07
+
```

Comment:
Is the app being used on any sites yet?

Reply:
It's not no

Reply:
Ok good just checking as if it was then any changes in this file wouldnt be applied